CMakeLists.txt

```diff
@@ -7,7 +7,7 @@
 #   - the gqcpy Python bindings to the C++ library (optional)
 
 cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
-project(GQCP VERSION 0.0.1 LANGUAGES CXX)
```

Comment:
I realized during the weekend that releasing a `v1.0.0` might not be the best idea yet. There still remain some old APIs (such as those related to AP1roG, response classes and orbital optimizers) that, in my opinion, should be improved  beforehand. Committing ourselves to a `v1.0.0` would mean throwing away the freedom we have to break APIs in `v0.x.y` versions.

I do realize that there is a minor inconsistency in our versioning at the moment, so perhaps we could refer to the 'paper' release as `v0.1.0` and from then on rigorously use the semantic versioning system?

Reply:
👍